lib/screens/bluetooth_screen.dart

```diff
@@ -18,7 +18,8 @@ class _BluetoothScreenState extends State<BluetoothScreen> {
     Bluetooth bluetooth = Bluetooth();
     bluetooth.findDevices().then((res) {
       bluetooth.connectToDevice();
-      Fluttertoast.showToast(msg: 'Connecting to device. This may take a minute.');
+      Fluttertoast.showToast(
+          msg: 'Connecting to device. This may take a minute.');
```

Comment:
Perhaps a scroll wheel that shows the app is still working could be added? 

What happens if the connection fails?

Reply:
None of the bluetooth connection logic is finished, all this changes is the linting